docs/tabs.md

```diff
@@ -83,3 +90,5 @@ Method                        | Description
 `size(string $value)`         | Size of the tabs list.                                                       | `''`
 `alignment(string $value)`    | Alignment of the tabs list.                                                  | `''`
 `style(string $value)`        | Style of the tabs list.                                                      | `''`
+`renderTabsContent(bool $value)` | Whether to render the `tabs-content` container and its content. | `true`
```

Comment:
Is there a use-case for not rendering tabs content if it's specified?

Reply:
Do you mean that? https://github.com/yiisoft/yii-bulma/blob/b90569981b668b31ab7ba5a2aa47eb4bf1700d9e/tests/TabsTest.php#L334

Reply:
Yes. Why do we want to specify content but not render it at the same time?

Reply:
Ready.